way_to_home/place/models.py

```diff
@@ -1 +1,68 @@
 """This module implements class that represents the place entity."""
+
+from user.models import User
+from django.db import models, IntegrityError
+from utils.abstract_models import AbstractModel
+
+
+class Place(AbstractModel):
+    """Model for place entity."""
+
+    longitude = models.DecimalField(decimal_places=6, max_digits=9)
+    latitude = models.DecimalField(decimal_places=6, max_digits=9)
+    name = models.CharField(max_length=64)
+    address = models.CharField(max_length=255)
+    stop_id = models.PositiveSmallIntegerField(null=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
```suggestion
    user = models.ForeignKey(User, on_delete=models.CASCADE)
```
Same situation as with name. From `route` model `user_id` is not necessary.

Reply:
done